parallel.cabal

```diff
@@ -12,9 +12,11 @@ category:       Control, Parallelism
 build-type:     Simple
 
 tested-with:
-  GHC == 9.10.0
+  -- Not supported yet
+  -- GHC == 9.12.0
```

Comment:
My fork supports 9.12.0, I think on the original Haskell CI the same is called 9.12.1.

Reply:
I am currently using the one from https://github.com/haskell-CI/haskell-ci @ master :confused:

Reply:
Ok, now used your branch.